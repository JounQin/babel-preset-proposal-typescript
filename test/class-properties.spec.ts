import { Counter } from './class-properties'
import { execute } from './helpers'

const proposal = 'class-properties'

test(proposal, () => {
  expect(() => execute(proposal)).toThrowErrorMatchingSnapshot()
  const counter = new Counter()
  expect(counter.getCount()).toEqual(0)
  expect(counter.increment().getCount()).toEqual(1)
})
