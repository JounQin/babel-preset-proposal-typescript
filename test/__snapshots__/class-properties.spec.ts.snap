// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`class-properties 1`] = `"Unexpected token ';'"`;
